auth/src/main/res/values-de-rAT/strings.xml

```diff
@@ -1,4 +1,4 @@
-<resources xmlns:tools="http://schemas.android.com/tools" xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
```

Comment:
Can we find a way to remove all these changes from the PR diff?  135 changed files makes me uncomfortable.  If you have to leave that one unused string in there, that's fine.

Reply:
Yeah, I need to get rid of the string to make lint happy, though the rest was IntelliJ's doing. 😆 Time to commit stuff from the CLI! 😢